### DISCUSSION TITLE
How do I command Airmar DST200 to send PGN 65410?

### DISCUSSION THREAD

**Discussion by keesverruijt** [canboat/canboat#287] [Show and tell]
Creating a separate issue for a separate question as posed by @mglonnro:

> Sorry if/when this is the wrong place to post this, but I'm trying my best to send 126208 to a DST800 (with actisense-serial), but can't get it to work. The idea is to get the DST800 to report the voltage it sees using PGN 65410.

Try this, with the latest master git commit:

    request-group-function 36 5 65410 2000 1=87 3=04 

where 36 is the Airmar's address, 5 is the prio, 65410 is the requested PGN, 2000 is the interval in milliseconds, 1=87 and 3=04 form the "Airmar specific" fields that according to the manual must be set.

I don't have a DST200 in my network right now so I can't test this.

I run the n2kd program so I send this using:

       request-group-function 36 5 65410 2000 1=87 3=04 | nc -W 1 localhost 2598

(assuming you're running this on the same system)

**Comment by mairas** [comment]
This Signal K plugin might be of some assistance: https://github.com/mairas/signalk-dst800-calibration-plugin

In the plugin, a DST800 is presuaded to output PGN 65409 for speed calibration purposes. The code is quite ugly but might give you ideas.
